safety_shield/include/safety_shield/long_term_traj.h

```diff
@@ -71,6 +73,25 @@ class LongTermTraj {
    */
   int starting_index_;
 
+  /**
+   * @brief maximum cartesian acceleration of robot joints (+ end effector!)
+   *
+   * alpha_i_.size() = nb_joints_ + 1
+   * TODO: Calculate this as overapproximation.
+   */
+  std::vector<double> alpha_i_;
+
+  /**
+   * @brief LTT-maximum of maximum cartesian velocity
+   */
+  double ltt_maximum_;
```

Comment:
The naming and function description do not really match.
Maybe rename to `ltt_max_vel_`?